doc/specs/staffPage.md

```diff
@@ -0,0 +1,152 @@
+# Staff Page
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
```

Comment:
I will make a PR removing all `4. Data model` from toc of existing spec since it was decided to separate what can be done by evryone from what need some more precise understanding of dev.